jdbc/src/test/java/nextstep/jdbc/JdbcTemplateTest.java

```diff
@@ -1,5 +1,123 @@
+        } catch (final SQLException e) {
+            throw new DataAccessException(e);
+        }
+
+    }
+
+    @Test
+    void update() {
+        //given
+        //when
+        String sql = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(sql, preparedStatement -> {
+            preparedStatement.setString(1, "account");
+            preparedStatement.setString(2, "password");
+            preparedStatement.setString(3, "email");
+        });
+
+        //then
+        RowMapper<Map<String, Object>> rowMapper = (resultSet, rowNum) -> Map.of(
+                "id", resultSet.getLong("id"),
+                "account", resultSet.getString("account"),
+                "password", resultSet.getString("password"),
+                "email", resultSet.getString("email")
+        );
+        String selectSql = "select * from users";
+        final List<Map<String, Object>> result = jdbcTemplate.query(selectSql, rowMapper);
+        assertSoftly(softly -> {
+            softly.assertThat(result).hasSize(1);
+            softly.assertThat(result.get(0).get("account")).isEqualTo("account");
+            softly.assertThat(result.get(0).get("password")).isEqualTo("password");
+            softly.assertThat(result.get(0).get("email")).isEqualTo("email");
+        });
+    }
+
+    @Test
+    public void queryList() {
+        //given
+        String sql = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(sql, preparedStatement -> {
+            preparedStatement.setString(1, "account");
+            preparedStatement.setString(2, "password");
+            preparedStatement.setString(3, "email");
+        });
+        jdbcTemplate.update(sql, preparedStatement -> {
+            preparedStatement.setString(1, "account2");
+            preparedStatement.setString(2, "password2");
+            preparedStatement.setString(3, "email2");
+        });
+
+        //when
+        String selectSql = "select * from users";
+        final List<Map<String, Object>> result = jdbcTemplate.query(selectSql, (resultSet, rowNum) -> Map.of(
+                "id", resultSet.getLong("id"),
+                "account", resultSet.getString("account"),
+                "password", resultSet.getString("password"),
+                "email", resultSet.getString("email")
+        ));
+
+        //then
+        assertSoftly(softly -> {
+            softly.assertThat(result).hasSize(2);
+        });
+    }
+
+    @Test
+    @DisplayName("가변인자를 이용한 쿼리 테스트")
+    void vargsQuery() {
+        //given
+        String sql = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(sql, preparedStatement -> {
+            preparedStatement.setString(1, "account");
+            preparedStatement.setString(2, "password");
+            preparedStatement.setString(3, "email");
+        });
+
```

Comment:
가변인자로 나머지도 리팩토링한다면 아래와 같이 사용할 수 있을 것 같아요!
```suggestion
jdbcTemplate.update(sql, "account", "password", "email");
```